library/src/main/java/com/novoda/downloadmanager/domain/DownloadFile.java

```diff
@@ -12,15 +12,13 @@
     private final long totalSize;
     private final String localUri;
     private final FileStatus fileStatus;
-    private final String fileIdentifier;
```

Comment:
Deffo need to keep the individual file identifiers. How else would you know what the individual files are? File uris are not ids!